tests/embedding/test_image_animation.py

```diff
@@ -0,0 +1,106 @@
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Image and animation export tests"""
+
+import os
+
+import pytest
+
+try:
+    from ansys.mechanical.core import global_variables
+except:
+    # No embedding - this import breaks test collection
+    global_variables = {}
+
+from .test_qk_eng_wb2 import get_assets_folder
+
+
+@pytest.mark.embedding
+def test_image_animation(printer, selection, embedded_app):
+    """Tests to check image and animation exports
+
+    ref:  Mechanical/QK_ENG_WB2/QK_ENG_WB2_005
+    """
+    globals().update(global_variables(embedded_app, True))
+    Model.AddStaticStructuralAnalysis()
+    geometry_file = os.path.join(get_assets_folder(), "Eng157.x_t")
+    geometry_import = Model.GeometryImportGroup.AddGeometryImport()
+    geometry_import.Import(geometry_file)
+    printer("Running test")
+
+    ExtAPI.Application.ActiveUnitSystem = MechanicalUnitSystem.StandardBIN
+    MODEL = Model
+    MODEL.Geometry.ElementControl = ElementControl.Manual
+    STAT_STRUC = MODEL.Analyses[0]
+    selection.UpdateSelection(ExtAPI, [26], SelectionTypeEnum.GeometryEntities)
+    FIX_SUP = STAT_STRUC.AddFixedSupport()
+    selection.UpdateSelection(ExtAPI, [25], SelectionTypeEnum.GeometryEntities)
+    FRC = STAT_STRUC.AddForce()
+    FRC.DefineBy = LoadDefineBy.Components
+    FRC.ZComponent.Output.SetDiscreteValue(0, Quantity("-1 [lbf]"))
+    ExtAPI.SelectionManager.ClearSelection()
+    DIR_DEF_STAT_STRUC = STAT_STRUC.Solution.AddDirectionalDeformation()
+    STAT_STRUC.Solution.Solve(True)
+    assert STAT_STRUC.Solution.ObjectState == ObjectState.Solved
+
+    Tree.Activate([DIR_DEF_STAT_STRUC])
+    ExtAPI.Graphics.Camera.SetFit()
+    settings_720p = Ansys.Mechanical.Graphics.GraphicsImageExportSettings()
+    settings_720p.Resolution = GraphicsResolutionType.EnhancedResolution
+    settings_720p.Background = GraphicsBackgroundType.White
+    settings_720p.Width = 1280
+    settings_720p.Height = 720
+    image_formats = ["PNG", "JPG", "BMP"]
+    for image_format in image_formats:
+        printer(f"{image_format} image export")
+        image_export_format = getattr(
+            Ansys.Mechanical.DataModel.Enums.GraphicsImageExportFormat, image_format
+        )
+        image_file = os.path.join(os.getcwd(), f"image.{image_export_format}")
+        ExtAPI.Graphics.ExportImage(image_file, image_export_format, settings_720p)
+        assert os.path.isfile(image_file)
+        with open(image_file, "rb") as file:
+            try:
+                file.read()
+            except Exception as e:
+                assert False, f"Failed to read file {image_file}: {e}"
+        os.remove(image_file)
+
+    animation_formats = ["GIF", "AVI", "MP4", "WMV"]
+    settings_720p = Ansys.Mechanical.Graphics.AnimationExportSettings()
+    settings_720p.Width = 1280
+    settings_720p.Height = 720
+    for animation_format in animation_formats:
```

Comment:
might be better to parameterize the test than have a loop here

Reply:
@dipinknair it is ok to open the mechdb multiple times. That is not expensive for a reasonably sized model.